lib/template/lib/tasks/rubocop.rake

```diff
@@ -1,6 +1,7 @@
+if Gem.loaded_specs.has_key?("rubocop-rspec")
```

Comment:
Yeah, I spent a while trying to find an alternative way for that. The `rescue LoadError` makes me cringe.